src/components/DownloadButton/DownloadButton.tsx

```diff
@@ -0,0 +1,84 @@
+import { memo, useCallback, type FC } from 'react'
+import { useAdvancedUserAgentData } from '@dcl/hooks'
+import { DownloadButton as DCLDownloadButton } from 'decentraland-ui2/dist/components/DownloadButton/DownloadButton'
+import { Box } from 'decentraland-ui2'
+import appleLogo from '../../assets/apple-logo.svg'
+import windowsLogo from '../../assets/windows-logo.svg'
+import { Events, useAnalytics } from '../../hooks/useAnalytics'
+import styles from './DownloadButton.module.css'
+
+const DOWNLOAD_URL_BASE = 'https://explorer-artifacts.decentraland.org/launcher/dcl/'
+
+enum OS {
+  MACOS = 'macos',
+  WINDOWS = 'windows'
+}
+
+enum ARCH {
+  ARM64 = 'arm64',
+  X64 = 'x64'
+}
+
+const DOWNLOAD_URLS = {
```

Comment:
all this config is already available on ui2, can we export them from there and use it here so we avoid having duplicates ?

Reply:
https://github.com/decentraland/ui2/blob/master/src/modules/cdnReleases.ts

Reply:
good catch, updated references!